src/libostree/ostree-repo-static-delta-compilation.c

```diff
@@ -1003,11 +1003,7 @@ generate_delta_lowlatency (OstreeRepo                       *repo,
       if (!check_object_world_readable (repo, from_checksum, &from_world_readable, cancellable, error))
         goto out;
       if (!from_world_readable)
-        {
-          g_hash_table_iter_steal (&hashiter);
-          g_hash_table_add (new_reachable_regfile_content, (char*)from_checksum);
-          continue;
-        }
+        continue;
```

Comment:
Took me a couple reads through this code again, but I agree this is right. There's no reason you'd want to include the old object in the delta. The rest of the code will DTRT as long as this unreadable object is kept out of the bsdiffs and rollsums to that the old object is not required on the pull side when applying the delta.